### ISSUE TITLE
add exit intent example

### ISSUE THREAD

**Issue by CypressCecelia** [cypress-io/cypress-example-kitchensink#453]
Add example for testing a modal or event that occurs when a user signals intent to exit the page by leaving the window with their mouse.

Need to create an example to test against, sample code below.

```
describe("Exit Intent test", function () {
it("should display modal on mouse exit", function () {
cy.visit("https://the-internet.herokuapp.com/exit_intent");
// Mouse into the window
cy.get("#flash-messages").trigger("mouseover");
// Mouse out of the window
cy.get("#flash-messages").trigger("mouseleave");
// This passes because the modal appears
cy.get(".modal");
});
});
```

**Comment by praveenpandey02** [comment]
Even I have having some issues to automate this scenario. This is one of the many things that I have tried so far but no success:

```
cy.get("body").trigger("mousemove, {clientX: 505, clientY: 100}");
cy.get(".modal").should("be.visible");

```

**Comment by praveenpandey02** [comment]
Hi @filiphric. Can you please advice how to deal with this case? I have already tried it but to no success.

**Comment by filiphric** [comment]
@praveenpandey02 do you have a page that you want to test this scenario on? Usually the leave intent is just some event listener on invisible element, so getting the right element and using `.trigger()` command should do the job. If not, it would really help to have the particular page available.

**Comment by praveenpandey02** [comment]
Yes @filiphric. This is the page I am trying to test: https://the-internet.herokuapp.com/exit_intent 
From Developer Tools, I also found that `mouseleave` is the event listener. So I am doing this:
`cy.get("body").trigger('mouseleave');
cy.get(".modal").should("be.visible");`

It doesn't work. I get this:
![image](https://user-images.githubusercontent.com/75299617/151934025-0574fd3b-e7c6-478e-b5d4-3b7dda967524.png)

**Comment by praveenpandey02** [comment]
Your solution worked like a charm! I am still wondering why `trigger('mouseleave')` did not work but `trigger('mouseleave', 0, 0)` worked 🤔
